cmd/api/src/model/appcfg/flag.go

```diff
@@ -76,10 +76,10 @@ func AvailableFlags() FeatureFlagSet {
 		},
 		FeatureAdcs: {
 			Key:           FeatureAdcs,
-			Name:          "Enable collection and processing of Active Directory Certificate Services Data",
-			Description:   "Enables the ability to collect, analyze, and explore Active Directory Certificate Services data and previews new attack paths.",
+			Name:          "Enable collection and processing of ADCS data",
+			Description:   "Enables the ability to analyze ADCS data for creating ADCS attack paths",
 			Enabled:       false,
-			UserUpdatable: true,
+			UserUpdatable: false,
```

Comment:
Were these changes intended? Or are we removing the ability for users to update the feature flag with the release of ESC3?

Reply:
Nope, it was a bad merge